simulator/src/sequential/JKflipFlop.js

```
@@ -66,7 +66,9 @@ export default class JKflipFlop extends CircuitElement {
      * in the clock. masterState = this.J when no change in clock.
      */
     resolve() {
-        if (this.reset.value == 1) {
+        if (this.preset.value == 1) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)